integration-test/src/test/resources/trino-ci-testset/testsets/jdbc-mysql/00002_alter_table.sql

```diff
@@ -18,7 +18,10 @@ show create table "test.gt_mysql".gt_db1.tb01;
 alter table "test.gt_mysql".gt_db1.tb01 alter column salary set data type bigint;
 show create table "test.gt_mysql".gt_db1.tb01;
 
```

Comment:
We should not remove  the `Alter table add column with null` test case.

Reply:
Fixed